imagery/i.atcorr/computations.cpp

```diff
@@ -444,7 +444,7 @@ void os(const double tamoy, const double trmoy, const double pizmoy,
     if (ntp == (snt - 1)) {
         /* compute position of the plane layer */
         double taup = tap + trp;
-        iplane = -1;
+        iplane = 0;
         for (int i = 0; i <= ntp; i++)
             if (taup >= h[i])
                 iplane = i;
```

Comment:
```suggestion
        iplane = -1;
        for (int i = 0; i <= ntp; i++) {
            if (taup >= h[i])
                iplane = i;
        }
        if (iplane == -1) {
            G_fatal_error(_("Position of the plane layer could not be determined"));
        }
```

Reply:
Made some very minor changes to accommodate clang-format in a later commit.